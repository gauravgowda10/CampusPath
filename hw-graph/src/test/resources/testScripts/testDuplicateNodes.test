# Tests that adding the same nodes multiple times does not create two of the same nodes in the graph

# Create graph
CreateGraph duplicateNodes

# Add nodes to duplicateNodes
AddNode duplicateNodes x
AddNode duplicateNodes y
AddNode duplicateNodes z

# Add Duplicate nodes to duplicateNodes
AddNode duplicateNodes x
AddNode duplicateNodes y
AddNode duplicateNodes z

# List nodes and the outgoing edges
ListNodes duplicateNodes