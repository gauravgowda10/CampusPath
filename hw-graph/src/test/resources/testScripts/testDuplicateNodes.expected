# Tests that adding the same nodes multiple times does not create two of the same nodes in the graph

# Create graph
created graph duplicateNodes

# Add nodes to duplicateNodes
added node x to duplicateNodes
added node y to duplicateNodes
added node z to duplicateNodes

# Add Duplicate nodes to duplicateNodes
added node x to duplicateNodes
added node y to duplicateNodes
added node z to duplicateNodes

# List nodes and the outgoing edges
duplicateNodes contains: x y z